examples/powershell/vro/template/powercli/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM photon:3.0
+ENV TERM linux
+
+WORKDIR /root
+
+# Set terminal. If we don't do this, weird readline things happen.
+RUN echo "/usr/bin/pwsh" >> /etc/shells && \
+    echo "/bin/pwsh" >> /etc/shells && \
+    tdnf install -y powershell-6.2.3-1.ph3 unzip && \
+    pwsh -c "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted" && \
+    find / -name "net45" | xargs rm -rf && \
+    tdnf erase -y unzip && \
+    tdnf clean all
+
+RUN mkdir -p /home/app
+USER root
```

Comment:
IMHO not required to be set  (also below)

Reply:
Ah. This was merely copy/paste from Alex original templates :)